src/components/ScyllaCloud.vue

```diff
@@ -59,7 +67,7 @@
           </table>
         </div>
       </div>
-      <div class="tech-specs">
+      <div v-if="!hideSpecs" class="tech-specs">
```

Comment:
use `v-show`

Reply:
makes no sense to render it and hide it if the prop is set to hide the specs for this component instance (won't be switched back and forth shown during its lifetime in our use case)